lib/src/domain/usecase/implementation/fetch_genres_usecase.dart

```diff
@@ -0,0 +1,11 @@
+import '../../../data/repository/genres_repository.dart';
+import '../genres_usecase_interface.dart';
+
+class FetchGenresUsecase implements GenresUseCase<void, void> {
```

Comment:
Why this use case is defined <void,void>?

Reply:
I use this case only to load the genres at the start of the app